scripts/genertate/calculate_metrics.py

```diff
@@ -0,0 +1,97 @@
+import re
+import argparse
+import sacrebleu
+from collections import Counter
+import operator
+import numpy as np
+from scipy import stats
+import os
+import random
+from gluonnlp.base import get_model_zoo_home_dir, get_repo_model_zoo_url, get_model_zoo_checksum_dir
+from gluonnlp.utils.misc import load_checksum_stats, download
+from gluonnlp.data.tokenizers import HuggingFaceByteBPETokenizer
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='Calculate metrics for the generated sentences')
+    parser.add_argument('--generated_file', type=str, required=True, help='Model name')
```

Comment:
You may just use `--file`, because these scores are also applicable to non-machine generated corpus.